src/dbt/kipptaf/models/deanslist/intermediate/int_deanslist__behavior_reporting_scaffold.sql

```diff
@@ -0,0 +1,127 @@
+with
+    behavior_categories as (
```

Comment:
@cbini judgement call - I really don't want to create another lookup table to maintain, but is that maybe the right move for this CTE?

Reply:
is this likely to change?

Reply:
yes, but hopefully only before the year starts

Reply:
doesn't seem big enough to merit a lookup table, but if it becomes a problem, that's a paddlin